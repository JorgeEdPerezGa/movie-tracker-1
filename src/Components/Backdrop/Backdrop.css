.backdrop {
  margin-top: 80px;
  width: 100%;
  height: auto;
  opacity: 0.15; }

.backdrop-container {
  position: relative; }

h3 {
  font-family: 'Comfortaa', cursive;
  font-weight: 100;
  font-size: 50px;
  margin: 0;
  color: #817df1;
  position: absolute;
  text-align: center;
  top: 40%;
  width: 100%;
  position: absolute; }

span {
  font-family: 'Comfortaa', cursive;
  color: white; }
