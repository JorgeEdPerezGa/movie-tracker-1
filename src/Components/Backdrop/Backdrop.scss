.backdrop {
  margin-top: 80px;
  width: 100%;
  height: auto;
  opacity: 0.1;

}

.backdrop-container {
  position: relative;
}

h3 {
  font-family: 'Comfortaa', cursive;
  font-weight: 100;
  font-size: 92px;
  margin: 0;
  color: #817df1;
  position: absolute;
  text-align: center;
  top: 35%;
  width: 100%;
  position: absolute;
}

span {
  font-family: 'Comfortaa', cursive;
  color: white;
}
