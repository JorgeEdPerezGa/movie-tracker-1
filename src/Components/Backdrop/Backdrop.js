import React from 'react';
import './Backdrop.css';
import { browserHistory } from 'react-router-dom'

const Backdrop = (props) => {
  // let number = Math.floor(Math.random() * (20) * 1);
  // console.log([props.backdrop]);
  // const backdrop = props.backdrop[number]

  return (
    // let number = await Math.floor(Math.random() * (20) * 1)
    <div className = 'backdrop-container'>
    <div className = 'backdrop-gradient'></div>
      <img className = 'backdrop' src={`https://image.tmdb.org/t/p/w500${props.backdrop}`} />
    </div>
  )
}

export default Backdrop;
