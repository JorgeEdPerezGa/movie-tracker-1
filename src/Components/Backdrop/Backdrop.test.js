import React from 'react';
import { shallow } from 'enzyme';
import Backdrop from './Backdrop';

let renderedBackdrop;
beforeEach(() => {
   renderedBackdrop = shallow(<Backdrop />);
});


describe('Backdrop Test', () => {

  it('should be defined', () => {


    expect(renderedBackdrop).toBeDefined();
  });


  it('should match snapshot', () => {
    expect(renderedBackdrop).toMatchSnapshot();
  })
})
