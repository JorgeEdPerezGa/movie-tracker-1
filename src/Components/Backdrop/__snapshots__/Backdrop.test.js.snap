// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Backdrop Test should match snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Backdrop />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h3>
          MOVIE
          <span>
             TRACKER
          </span>
        </h3>,
        <img
          className="backdrop"
          src="https://image.tmdb.org/t/p/w500undefined"
        />,
      ],
      "className": "backdrop-container",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "MOVIE",
            <span>
               TRACKER
            </span>,
          ],
        },
        "ref": null,
        "rendered": Array [
          "MOVIE",
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": " TRACKER",
            },
            "ref": null,
            "rendered": " TRACKER",
            "type": "span",
          },
        ],
        "type": "h3",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "backdrop",
          "src": "https://image.tmdb.org/t/p/w500undefined",
        },
        "ref": null,
        "rendered": null,
        "type": "img",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h3>
            MOVIE
            <span>
               TRACKER
            </span>
          </h3>,
          <img
            className="backdrop"
            src="https://image.tmdb.org/t/p/w500undefined"
          />,
        ],
        "className": "backdrop-container",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "MOVIE",
              <span>
                 TRACKER
              </span>,
            ],
          },
          "ref": null,
          "rendered": Array [
            "MOVIE",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": " TRACKER",
              },
              "ref": null,
              "rendered": " TRACKER",
              "type": "span",
            },
          ],
          "type": "h3",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "backdrop",
            "src": "https://image.tmdb.org/t/p/w500undefined",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
